docs/configuration.md

```diff
@@ -29,12 +29,46 @@ BARTENDER_ENVIRONMENT="dev"
 You can read more about BaseSettings class here:
 <https://pydantic-docs.helpmanual.io/usage/settings/>
 
+## Authentication
+
+The bartender web service uses [JWT tokens](https://jwt.io/) for authentication.
+
+The tokens should use the RS256 algorithm,
+which requires a public and private RSA key pair.
+A key pair can be generated with
+
+```bash
+openssl genpkey -algorithm RSA -out private_key.pem -pkeyopt rsa_keygen_bits:2048
+openssl rsa -pubout -in private_key.pem -out public_key.pem
+```
+
+The private key of the RSA key pair is used to generate a token in
+an another web application or with the `bartender generate-token` command.
+
+The public key of the RSA key pair is used to verify that the token comes
+from a trusted source.
+The public key file location is `public_key.pem`
+or value of `BARTENDER_PUBLIC_KEY` environment variable.
```

Comment:
If my understanding is correct, in this case, the "sender" should be the owner of the keypair. The sender uses their private key to create a token and the "recipient" uses the senders' public key to verify that it could only have been the sender (i.e. the only one knowing their private key) to have generated the token. Correct? 

So does that mean that bartender could potentially store multiple public keys for various senders?

Reply:
Create https://github.com/i-VRESSE/bartender/issues/75 for multiple public keys. Will implement when we have a concrete use case.

Reply:
> If my understanding is correct, in this case, the "sender" should be the owner of the keypair. The sender uses their private key to create a token and the "recipient" uses the senders' public key to verify that it could only have been the sender (i.e. the only one knowing their private key) to have generated the token. Correct?
> 
Yep, correct. That is how https://en.wikipedia.org/wiki/Digital_signature works. The signer owns the private key and gives a public key to the verifier.

> So does that mean that bartender could potentially store multiple public keys for various senders?